internals/secrethub/run.go

```diff
@@ -422,6 +424,48 @@ func ReadEnvFile(filepath string, varReader tpl.VariableReader, parser tpl.Parse
 	}, nil
 }
 
+// OsEnv is an environment with secrets configured with the
+// secrethub:// syntax in the os environment variables.
+type OsEnv struct {
+	envVars map[string]string
+}
+
+// NewOsEnvSource returns an environment with secrets configured in the
+// os environment with the secrethub:// syntax.
+func NewOsEnvSource(osEnv map[string]string) *OsEnv {
+	envVars := make(map[string]string)
+	for key, value := range osEnv {
+		if strings.HasPrefix(value, "secrethub://") {
+			envVars[key] = strings.TrimPrefix(value, "secrethub://")
```

Comment:
I'd move the `secrethub://` value to constant. Duplicate constants are evil :grimacing:

Reply:
Sure thing, fixed it!